app.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+var flash = require('connect-flash');
+var session = require('express-session');
+var passport = require('passport');
+var flash = require('connect-flash');
+var cookieParser = require('cookie-parser');
+var morgan = require('morgan');
+const app = express();
+
+//Middleware
+//ejs
+app.set("view engine", "ejs");
+app.use(express.static(__dirname + '/public'));
+//Morgan
+app.use(morgan('dev'));
+//Cookie Parser
+app.use(cookieParser());
+require('./config/passport')(passport);
```

Comment:
all imports are preferred to be at the top of the file, non-library imports below library imports. I'd change to this example:
```const passportConfig = require('./config/passport');
passportConfig(passport);```